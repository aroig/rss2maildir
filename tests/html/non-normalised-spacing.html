<h1>This  has some	odd    spacing</h1>
<p>It's really  great    and hopefully  shouldn't
be

too bad over
all
</p>
