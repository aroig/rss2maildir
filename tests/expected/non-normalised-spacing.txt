This has some odd spacing
=========================

It's really great and hopefully shouldn't be too bad over all
